options/locale/locale_en-US.ini

```diff
@@ -1542,7 +1542,7 @@ file_too_big = File size ({{filesize}} MB) exceeds the maximum size of ({{maxFil
 remove_file = Remove file
 
 [signup]
-agreement = By creating an account on this website, you affirm that you have read and understood the <a href="/Door43/en-legal/wiki/Terms-%26-Conditions" target="_blank">Terms & Conditions</a> and agree to:<ul><li>Only add content to this website that does not infringe on someone else's copyright.</li><li>Only add content to this website that does not conflict with the <a href="/Door43/en-legal/wiki/Statement-of-Faith" target="_blank">Statement of Faith</a>.</li><li>Release your contributions to the content on this website under a <a href="/Door43/en-legal/wiki/Copyrights-%26-Licensing" target="_blank">Creative Commons Attribution-ShareAlike 4.0 International License.</a></li>
+agreement = By creating an account on this website, you affirm that you have read and understood the <a href="https://door43.org/en/legal" target="_blank">Terms & Conditions</a> and agree to:<ul><li>Only add content to this website that does not infringe on someone else's copyright.</li><li>Only add content to this website that does not conflict with the <a href="https://door43.org/en/statement-of-faith" target="_blank">Statement of Faith</a>.</li><li>Release your contributions to the content on this website under a <a href="https://door43.org/en/copyrights-and-licensing" target="_blank">Creative Commons Attribution-ShareAlike 4.0 International License.</a></li>
```

Comment:
The copyright link here is not correct, the "s" after copyright should be removed.  Also, the links should all end with a / to avoid an unnecessary redirect.

Reply:
Done